weather/ngsild-payloads/weatherAlert.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+	"id": "urn:ngsi-ld:WeatherAlert:Menton",
+	"type": "WeatherAlert",
+	"weatherRisk": [
+		{
+			"type": "Property",
+			"value": "Des averses d'intensités modérées",
+			"riskType": {
+				"type": "Property",
+				"value": "Pluie"
+			},
+			"riskColor": {
+				"type": "Property",
+				"value": "Jaune"
+			},
```

Comment:
a bit misleading. in the orginal API, risk color is a color code. and "Jaune" is the risk name.